src/ImageSharp/Formats/Webp/BitWriter/Vp8BitWriter.cs

```diff
@@ -592,7 +592,7 @@ private void CodeIntraModes()
 
     private void WriteVp8Header(Stream stream, uint size)
     {
-        Span<byte> buf = stackalloc byte[WebpConstants.TagSize];
+        Span<byte> buf = byte[WebpConstants.TagSize];
```

Comment:
I assume this is an accidental change?

Reply:
OOPS! It was supposed to be a copy and ended up being a move.

Punishment for being lazy. 😄